src/components/List.js

```diff
@@ -79,6 +80,42 @@ const List = ({ token }) => {
     return daysUntilNextPurchase > 0 ? daysUntilNextPurchase : 0;
   };
 
+  const sortedList = docs.sort((a, b) => {
```

Comment:
Here's a good resource we found on the [sort function](https://www.javascripttutorial.net/javascript-array-sort/)

Reply:
I really like that you linked to the documentation here! It made it really easy for me to quickly review how the sort method works! Thank you.